### PR TITLE
init: finish task-3

### PR DESCRIPTION
**Pull request by izzetbek**
1. Serverless is configured
2. getProductsList  - https://gblhr1x62d.execute-api.eu-west-1.amazonaws.com/dev/products
3. getProductsById - https://gblhr1x62d.execute-api.eu-west-1.amazonaws.com/dev/products/7567ec4b-b10c-48c5-9345-fc73c48a80a0
4. Front-end: https://d3hmyqwjwvay0w.cloudfront.net/
5. Async/await is used in lambda functions
6. ES6 modules are used for product-service implementation
7. Webpack is configured for product-service
8. SWAGGER documentation is created for product-service
9. Lambda handlers are covered by basic UNIT tests (NO infrastructure logic is needed to be covered) (You may use JEST)
10. Lambda handlers (getProductsList, getProductsById) code is written not in 1 single module (file) and separated in codebase.
11. Main error scenarious are handled by API ("Product not found" error, try catch blocks are used in lambda handlers).

### CODE REVIEW

**Comment by andreikasmykou** [comment]
Please edit PR comment according to the task description. Thank you

**Comment by izzetbek** [comment]
> Please edit PR comment according to the task description. Thank you

Updated, thank you.

File: product-service/Products-Swagger20.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "description": "",
+            "schema": {
+              "type": "object"
+            },
+            "headers": {}
+          }
+        }
+      }
+    },
+    "/products/7567ec4b-b10c-48c5-9345-fc73c48a80a0": {
```
Comment (bdvx) [inline product-service/Products-Swagger20.json]: I guess we need to include here smth like "/products/{productId}" with description, and not hardcoded id

Comment (izzetbek) [reply product-service/Products-Swagger20.json]: Fixed, thank you

**Pull request merged**